integrations/flink/documentation.yaml

```diff
@@ -36,7 +36,7 @@ additional_prereq_info: |
     flinkVersion: v1_15
```

Comment:
Should we update all of the references to flink 1.15 to be 1.17 as well (if this change isn't backwards compatible)? If so, then let's also update it in the prometheus metadata too https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/integrations/flink/prometheus_metadata.yaml#L10

Reply:
Changes pushed, thanks for catching that!

Reply:
Doubling checking - should the image and flinkVersion values here be updated as well?

Reply:
Yes, it should, thank you!

Reply:
Great, thanks.